### ISSUE TITLE
New issue in post #3

### ISSUE THREAD

**Issue by niguarino** [American-Institutes-for-Research/EdSurvey#32]
I just added a question to "Transforming ECLSK 2011 data from "wide" to "long" format #3". I wanted to bring it to your attention here in case only new issues prompt your attention. Thank you.

**Comment by pdbailey0** [comment]
@niguarino yes, I see whenever someone posts. Can you please start a new issue that is just your post on that other thread? You'll notice that including a reference (as you did here) includes a note in both issues--and that is a good idea. I can then clean everything up, deleting issues and posts as needed.
